spec/rails_helper.rb

```diff
@@ -0,0 +1,71 @@
+# This file is copied to spec/ when you run 'rails generate rspec:install'
+ENV['RAILS_ENV'] ||= 'test'
+require File.expand_path('../../config/environment', __FILE__)
+# Prevent database truncation if the environment is production
+abort("The Rails environment is running in production mode!") if Rails.env.production?
+require 'spec_helper'
+require 'rspec/rails'
+require 'factory_girl_rails'
+require 'database_cleaner'
+
+# Add additional requires below this line. Rails is not loaded until this point!
+Dir[__dir__ + '/support/*.rb'].each { |file| require_relative file }
+Dir[__dir__ + '/support/**/*.rb'].each { |f| require f }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.